### PR TITLE
Add listHistory() on buckets

### PR DESCRIPTION
**Pull request by leplatrem**
* Wait for release of https://github.com/Kinto/kinto/issues/773
* Wait for release of https://github.com/Kinto/kinto/issues/774

@n1k0 r?

### CODE REVIEW

File: test/integration_test.js [hunk 1]
```diff
@@ -609,6 +609,72 @@ describe("Integration tests", function() {
         });
       });
 
+      describe(".listHistory()", () => {
+        it("should retrieve the list of history entries", () => {
+          return bucket.listHistory()
+            .then(({data}) => data.map(entry => entry.target.data.id).sort())
```
Comment (n1k0) [inline test/integration_test.js]: Out of cusriosity, why do we need to sort? Isn't the order predictable?

**Comment by n1k0** [comment]
Looks amazing! Now waiting for the other patches to land :)

**Comment by leplatrem** [comment]
Let's see if server 4.0.1 fixes it ;)

**Comment by n1k0** [comment]
\o/

**Pull request merged**